backend/src/graphql/resolvers/TableResolver.ts

```diff
@@ -220,6 +165,50 @@ export class TableResolver {
     return openTablesFromOpenMeetings(meetings)
   }
 
+  @Authorized()
+  @Mutation(() => Table)
+  async createTable(
+    // params: CreateGroupTableParams,
+    @Arg('name') name: string,
+    @Arg('isPublic') isPublic: boolean,
+    @Ctx() context: Context,
+    // eslint-disable-next-line type-graphql/wrong-decorator-signature
+    @Arg('userIds', () => [Int], { nullable: 'itemsAndList' }) // eslint-disable-next-line type-graphql/invalid-nullable-input-type
+    userIds?: number[] | null | undefined,
+  ): Promise<Table> {
+    const { user } = context
+    if (!user) throw new Error('User not found!')
+
+    // Get Meeting where user.id and userIds are Moderator
+    // Check if GroupMeeting already exist how to recognize the Meeting?
+    // if (user.meetingId) {
+    //   throw new Error('Meeting already exists!')
+    // }
+
+    const meetingID: string = await createMeetingID()
+
+    const dbMeeting = await prisma.meeting.create({
+      data: { meetingID, name, public: isPublic },
+    })
+
+    if (!userIds) {
+      userIds = []
+    }
+
+    if (!userIds.some((userId) => userId === user.id)) {
+      userIds.push(user.id)
+    }
+
+    if (userIds) {
```

Comment:
```suggestion
    if (userIds.length) {
```
an empty array is true